src/components/Wizards/RosaWizard/common/CopyInstruction.css

```
@@ -0,0 +1,4 @@
+.copy-instruction pre {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)